best_practices/controllers.rst

```diff
@@ -164,13 +177,14 @@ manually. In our application, we have this situation in ``CommentController``:
 
 .. code-block:: php
 
+    use Doctrine\ORM\EntityManagerInterface;
+
     /**
      * @Route("/comment/{postSlug}/new", name = "comment_new")
      */
-    public function newAction(Request $request, $postSlug)
+    public function newAction(Request $request, $postSlug, EntityManagerInterface $em)
```

Comment:
I talked with Nicolas about this about a month ago, we decided (for now) to recommend using `Controller`... it's less of a big change because we're not taking away the "escape hatch" of using the container as a service locator. We can (likely will) change that recommendation in the future as people get more accustomed to not using `$this->get()`.